packages/auth-providers/dbAuth/middleware/modules.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '@redwoodjs/vite/middleware' {
```

Comment:
@dac09 I was not sure of this, but seems to work to get TS to be happy and build